Sources/Perform/Perform.swift

```diff
@@ -3,5 +3,39 @@ import Result
 import UIKit
 
 extension UIViewController {
-  public static let perform: SignalProducer<String, NoError> = .init(value: "Hello, Perform")
+  public func perform<Destination: UIViewController>(segue: Segue<Destination>, configure: (Destination) -> Void) {
```

Comment:
Yeah I wondered about that. I suppose it would make sense for completeness, although I'm not sure whether it actually influences the behaviour of the segue in any way.

Here's what [the documentation](https://developer.apple.com/reference/uikit/uiviewcontroller/1621413-performsegue) says:

> `sender`: The object that you want to use to initiate the segue. This object is made available for informational purposes during the actual segue.

So given that the pattern matching is based on type only, I don't think the sender would be useful here.